plugins/orchestrator/src/components/next/WorkflowRunsTabContent.tsx

```diff
@@ -5,83 +5,73 @@ import { useAsync } from 'react-use';
 import { SelectItem, Table, TableColumn } from '@backstage/core-components';
 import { useApi, useRouteRef } from '@backstage/core-plugin-api';
 
-import { Box } from '@material-ui/core';
-import moment from 'moment';
+import { Box, makeStyles } from '@material-ui/core';
+
+import { WorkflowCategory } from '@janus-idp/backstage-plugin-orchestrator-common';
 
 import { orchestratorApiRef } from '../../api';
-import { workflowInstanceRouteRef } from '../../routes';
+import { nextWorkflowInstanceRouteRef } from '../../routes';
 import { ProcessInstanceStatus } from './ProcessInstanceStatus';
 import { StatusSelector } from './StatusSelector';
 import { TableExpandCollapse } from './TableExpandCollapse';
+import { firstLetterCapital } from './utils';
+import {
+  mapProcessInstanceToDetails,
+  WorkflowRunDetail,
+} from './WorkflowInstancePageContent';
 import { WorkflowSelector } from './WorkflowSelector';
 import { WrapperInfoCard } from './WrapperInfoCard';
 
-const DASHES = '---';
-interface Row {
-  id: string;
-  name: string;
-  workflow: string;
-  status: string;
-  started: string;
-  duration: string;
-  component: string;
-  parentInstanceId?: string;
-}
+const useStyles = makeStyles(_ => ({
+  link: {
+    color: '-webkit-link',
```

Comment:
Why do you need this?
(I'm asking because I did something similar in the `WorkflowsTable` and didn't need to override the theme)

Reply:
The link was there but in a regular font, not blue like expected